.changes/sql-uuid-type.md

```diff
@@ -0,0 +1,7 @@
+---
+"sql": patch
+---
+
+### Changes:
+- The `to_json` function in `postgres.rs` has been modified to include `UUID` as a supported type for decoding.
+- In the match statement for `v.type_info().name()`, the type `UUID` has been added to the list of supported types that decode into `JsonValue::String`.
```

Comment:
```suggestion
Added support for `UUID` columns to the postgres implementation.
```

While _I_ really appreciate descriptions like this. The implementation details typically don't matter to end-users (or from experience, just confuse them), so let's keep this more generic :)